easybuild/tools/include.py

```diff
@@ -72,9 +72,11 @@
 for subdir in subdirs:
     __path__ = pkgutil.extend_path(__path__, '%s.%s' % (__name__, subdir))
 
-del l, subdir, subdirs
+del subdir, subdirs
+if 'char' in dir():
+    del char
 
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
+__path__ = pkgutil.extend_path(__path__, __name__)
 """
```

Comment:
Could there be a reason why this is not used in the other code?

Reply:
The other code is literally a single line and the import is only used once. Here it is used multiple times